src/EA.Weee.Web/App_Start/CustomDateBinder.cs

```diff
@@ -32,7 +33,7 @@ public object BindModel(ControllerContext controllerContext, ModelBindingContext
             }
             catch (Exception)
             {
-                bindingContext.ModelState.AddModelError(bindingContext.ModelName, string.Format("\"{0}\" is invalid.", bindingContext.ModelName));
+                bindingContext.ModelState.AddModelError(bindingContext.ModelName, $"{bindingContext.ModelMetadata.DisplayName} is invalid");
```

Comment:
wondering whether the ModelMetadata.PropertyName to be included in the error too, up to you

Reply:
It turns out it displays "Start date is invalid" so looks ok

Reply:
alright